docs/manual/other-topics/upgrade-to-v6.md

```diff
@@ -46,6 +46,21 @@ This method now tests for equality with [`_.isEqual`](https://lodash.com/docs/4.
   await instance.save(); // will save
 ```
 
+#### Finder methods with empty attributes
```

Comment:
This should be treated as a bug fix, no need to mention this here. Apart from postgres it was not working, something like a type error.

When next release is pushed I will update the changelog along with other commits.

Reply:
Hmmm, if you say so :+1: